ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__aws-4.13-nightly-arm.yaml

```diff
@@ -21,7 +21,7 @@ releases:
     candidate:
       product: ocp
       stream: nightly
-      version: "4.14"
+      version: "4.13"
```

Comment:
@svetsa-rh I don't understand why the 'latest' release is need here?
And why only the arm64 test needs to specify the dependency OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE in the tests?

Reply:
@qiliRedHat That's because the default release images point to amd64 bits and since we want to point to ARM binaries, we need to specify override.